# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/cabextract-uclibc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- cabextract/fnmatch.c.orig	2005-01-22 06:16:14.000000000 +0000
+++ cabextract/fnmatch.c	2005-01-22 06:20:56.000000000 +0000
@@ -45,7 +45,6 @@
 
 #include <errno.h>
 #include <fnmatch.h>
-#include <safe-ctype.h>
 
 /* Comment out all this code if we are using the GNU C Library, and are not
    actually compiling the library itself.  This code is part of the GNU C
@@ -55,7 +54,7 @@
    program understand `configure --with-gnu-libc' and omit the object files,
    it is simpler to just do this in the source for each such file.  */
 
-#if defined (_LIBC) || !defined (__GNU_LIBRARY__)
+//#if defined (_LIBC) || !defined (__GNU_LIBRARY__)
 
 
 #if !defined(__GNU_LIBRARY__) && !defined(STDC_HEADERS)
@@ -65,7 +64,7 @@
 /* Match STRING against the filename pattern PATTERN, returning zero if
    it matches, nonzero if not.  */
 int
-fnmatch (pattern, string, flags)
+rpl_fnmatch (pattern, string, flags)
      const char *pattern;
      const char *string;
      int flags;
@@ -73,7 +72,7 @@
   register const char *p = pattern, *n = string;
   register unsigned char c;
 
-#define FOLD(c)	((flags & FNM_CASEFOLD) ? TOLOWER (c) : (c))
+#define FOLD(c)	((flags & FNM_CASEFOLD) ? tolower (c) : (c))
 
   while ((c = *p++) != '\0')
     {
@@ -220,4 +219,4 @@
   return FNM_NOMATCH;
 }
 
-#endif	/* _LIBC or not __GNU_LIBRARY__.  */
+//#endif	/* _LIBC or not __GNU_LIBRARY__.  */
